pansat/products/reanalysis/era5.py

```diff
@@ -47,17 +47,17 @@ def __init__(self, tsteps, levels, variables, domain=None, name=None):
         if not domain:
             self.domain = "global"
         else:
-            if domain[0] < -90 or domain[0] > 90 or domain[1] > 90 or domain[1] < -90:
-                raise Exception("Longitude values have to be between -90 and 90.")
-
             if (
-                domain[2] < -180
-                or domain[2] > 180
-                or domain[3] > 180
-                or domain[3] < -180
+                domain[0] < -180
+                or domain[0] > 180
+                or domain[1] > 180
+                or domain[1] < -180
             ):
                 raise Exception("Latitude values have to be between -180 and 180.")
 
+            if domain[2] < -90 or domain[2] > 90 or domain[3] > 90 or domain[3] < -90:
+                raise Exception("Longitude values have to be between -90 and 90.")
```

Comment:
and here 180 instead of 90 for lon?

Reply:
ah yes, thank you! sorry for the chaos!

Reply:
no problem! I myself always have to think twice with lat/lon